crates/ide-diagnostics/src/handlers/missing_fields.rs

```diff
@@ -55,6 +103,11 @@ fn fixes(ctx: &DiagnosticsContext<'_>, d: &hir::MissingFields) -> Option<Vec<Ass
 
     let root = ctx.sema.db.parse_or_expand(d.file)?;
 
+    let current_module = match &d.field_list_parent {
+        Either::Left(ptr) => ctx.sema.scope(ptr.to_node(&root).syntax()).unwrap().module(),
+        Either::Right(ptr) => ctx.sema.scope(ptr.to_node(&root).syntax()).unwrap().module(),
+    };
```

Comment:
We shouldn't be able to hit these if we are reporting diagnotics I think, but we should still avoid the unwraps nevertheless. Let's go with `ctx.sema.scope(ptr.to_node(&root).syntax()).map(|it| it.module())` and check in the loop if the module exists.